1-exercises/A-reading-list/script.js

```diff
@@ -1,7 +1,27 @@
 function readingList(books) {
-  // Write your code here...
+let divEl =document.getElementById("content");
+books.forEach((book) => {
+ let pEl = document.createElement("p");
+ let liEl  = document.createElement("li");
+ let imgEl  = document.createElement("img");
+ let ulEl = document.createElement("ul");
+ 
+    pEl.appendChild(
+      document.createTextNode(`"${book.title}" by ${book.author}`)
+    );
+    liEl.appendChild(pEl);
+    imgEl.setAttribute("src", book.bookCoverImage);
+    liEl.appendChild(imgEl);
+    ulEl.appendChild(liEl); 
+
+        if (book.alreadyRead) liEl.className = "green";
+    else liEl.className = "red";
+    content.appendChild(ulEl);
+  });
 }
 
```

Comment:
really nice solution Shimen :) you could look to use a code formatter to help tidy up the formatting of your code - a popular one is **Prettier - Code Formatter** (You can install it through vscode)